File: tests/fixtures/sample-js-project-il/code/src/legacy/uploader.ts
// Legacy file-upload module. ADR-003 moved uploads out of scope; this
// `src/legacy/` directory should have been deleted but still ships.
// IL-DRIFT: ForbiddenArtifact:legacy-uploader / forbidden.file-glob.src/legacy/**.present
export async function legacyUpload(_path: string): Promise<void> {
  throw new Error('legacy uploader has been retired');
}

File: tests/fixtures/sample-python-project-il/code/app/legacy/uploader.py
# Legacy file-upload module. ADR-003 moved uploads out of scope; this
# `app/legacy/` package should have been deleted but still ships.
# IL-DRIFT: ForbiddenArtifact:legacy-uploader / forbidden.file-glob.app/legacy/**.present


def legacy_upload(path: str) -> None:
    raise RuntimeError("legacy uploader has been retired")
